upcloud/client/client.go

```diff
@@ -184,9 +209,9 @@ func (c *Client) PerformRequest(request *http.Request) ([]byte, error) {
 
 // Returns the base URL to use for API requests
 func (c *Client) getBaseURL() string {
-	urlVersion, _ := semver.Make(DefaultAPIVersion)
+	urlVersion, _ := semver.Make(c.apiVersion)
```

Comment:
Should we check for errors here, in case the user has entered an invalid version? Panic should be fine, I guess, unless you want to add error handling here as well? Otherwise, it might be kind of unclear as `urlVersion` would (probably?) end up as "", and the error message then being kind of odd with just a faulty URL with no other indications of what went wrong?

Reply:
yes, an oversight.